### PR TITLE
Add symlinks under node_modules as part of projectRoots

### PR DESCRIPTION
**Pull request by harshil07**
Support symlinks under `node_modules` for all local-cli commands. PR https://github.com/facebook/react-native/pull/9009 only adds symlink support to the packager. 

But other cli commands like `react-native bundle` creates its own instance of packager that doesn't have symlinks as part of its project roots, which results in the bundler breaking since it cannot find modules that you have symlinked. 

This change ensures all `local-cli` commands add symlinks to its project roots.

Test plan (required)

1.  Create a symlink in node_modules (for instance use npm/yarn link)
2. Run `react-native bundle`.

### CODE REVIEW

File: local-cli/core/default.config.js [hunk 1]
```diff
@@ -22,6 +22,9 @@ const windows = require('./windows');
 const wrapCommands = require('./wrapCommands');
 const findPlugins = require('./findPlugins');
 
+const findSymlinksPaths = require('../util/findSymlinksPaths');
```
Comment (Kureev) [inline local-cli/core/default.config.js]: AFAIK it's `../server/findSymlinksPaths`

Comment (Kureev) [inline local-cli/core/default.config.js]: Can you please double-check if your code works, @harshil07?

Comment (harshil07) [inline local-cli/core/default.config.js]: it does work @Kureev . i moved the file to util as part of this PR given it was being used outside of "server" code.

image

Comment (Kureev) [inline local-cli/core/default.config.js]: Ah, I see. Missed this line

**Comment by Kureev** [comment]
@facebook-github-bot shipit

File: local-cli/server/server.js [hunk 2]
```diff
@@ -9,20 +9,15 @@
...
 
 /**
  * Starts the React Native Packager Server.
  */
 function server(argv, config, args) {
-  const roots = args.projectRoots.concat(args.root);
-  args.projectRoots = roots.concat(
-    findSymlinksPaths(NODE_MODULES, roots)
-  );
+  args.projectRoots.concat(args.root);
```
Comment (Kureev) [inline local-cli/server/server.js]: woah, how could I miss that

**Pull request closed without merging**